README.md

```diff
@@ -60,18 +64,27 @@ Check out all related information [here](GSSoC.md)
   - stanford_path_to_models_jar = <your_path_to_stanford-corenlp-x.x.x-models.jar>
   - javahome = <your_path_to_jdk_bin_java.exe>
 - Install dependencies from `requirements.txt` file. Run `pip install -r requirements.txt`
-- You're all set up, run the `init.py` file. `python init.py` 
-- It is recommended that you set this project up in a virtual environment to keep the dependencies separated and for easier debugging. Here's how you can do that - 
+- You're all set up, run the `init.py` file. `python init.py`
+- It is recommended that you set this project up in a virtual environment to keep the dependencies separated and for easier debugging. Here's how you can do that -
     1. [Python](https://realpython.com/python-virtual-environments-a-primer/#why-the-need-for-virtual-environments)
     2. [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
 
+#### 2. How to set me up Docker style?
+- Make sure you have Docker and Docker-Compose (refer pre-requisite steps for installation instructions)
```

Comment:
Add pre-requisites here, omit this redundant validation check.

Reply:
Removed validation check